cmd/cli/policy_check_pods.go

```diff
@@ -14,6 +14,7 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 
+	policyCheckHelper "github.com/openservicemesh/osm/pkg/cli"
```

Comment:
No need to alias this internal pkg. Use `cli.Xyz` to reference variables/functions in the `cli` pkg.

```suggestion
	"github.com/openservicemesh/osm/pkg/cli"
```